tests/dags/common/operators/test_postgres_result.py

```diff
@@ -0,0 +1,72 @@
+"""
+Based on:
+https://airflow.apache.org/docs/apache-airflow/stable/best-practices.html#unit-tests
+"""
+import datetime
+import os
+
+import pendulum
+import pytest
+from airflow import DAG
+from airflow.models import DagRun, TaskInstance
+from airflow.utils.session import create_session
+from airflow.utils.state import DagRunState, TaskInstanceState
+from airflow.utils.types import DagRunType
+from common.operators.postgres_result import PostgresResultOperator
+
+
+DATA_INTERVAL_START = pendulum.datetime(2021, 9, 13, tz="UTC")
+DATA_INTERVAL_END = DATA_INTERVAL_START + datetime.timedelta(days=1)
+
+TEST_DAG_ID = "test_postgres_result_dag"
+TEST_TASK_ID = "test_postgres_result_task"
+DB_CONN_ID = os.getenv("OPENLEDGER_CONN_ID", "postgres_openledger_testing")
+
+
+@pytest.fixture(autouse=True)
+def clean_db():
+    with create_session() as session:
+        session.query(DagRun).filter(DagRun.dag_id == TEST_DAG_ID).delete()
+        session.query(TaskInstance).filter(TaskInstance.dag_id == TEST_DAG_ID).delete()
+        yield
+        session.query(DagRun).filter(DagRun.dag_id == TEST_DAG_ID).delete()
+        session.query(TaskInstance).filter(TaskInstance.dag_id == TEST_DAG_ID).delete()
```

Comment:
Why does this `yield` and re-run the clean up?

Reply:
This might be overkill 😅 I wanted to catch the case where maybe a previous cleanup didn't run successfully or something, so it cleans up both before _and_ after the test. Do you think that's too much? Should it just clean up after the test is run?